Add `Resurrection Cup 2025: Registrations Now Open!` news post (#13238)
ppy/osu-wiki: news/2025/2025-05-11-resurrection-cup-2025-registrations-now-open.md

---
layout: post
title: "Resurrection Cup 2025: Registrations Now Open!"
date: 2025-05-11 12:00:00 +0000
---

The renaissance is upon us with Resurrection Cup returning for a 2025 iteration — with more bangers, more customs and a large focus on competitivity!

![](/wiki/Tournaments/RESC/2025/img/banner.jpg)

> [Resurrection Cup](https://osu.ppy.sh/community/forums/topics/20767782), or ResCup for short, is an annual open-rank Team Vs tournament that attempts to bring a breath of fresh air to the top-tier competitive tournament scene.

Reviewer: ```suggestion
[Resurrection Cup](https://osu.ppy.sh/community/forums/topics/20767782), or ResCup for short, is an annual open-rank Team VS tournament that attempts to bring a breath of fresh air to the top-tier competitive tournament scene.
```

## Original music and original maps

Over the past couple years, we have had the joy of sharing 40 originals with you. You can expect that trend to continue this year as well since, after the Qualifiers have finished, we will be releasing multiple original songs in every single mappool. If you thought last year was good, you bet we are going to surpass expectations this year as well, so be sure to tune in.

> If you haven't seen it yet, here is a crossfade of the upcoming *Wishes for Tomorrow* album which will include 22 of them!

Reviewer: this was announced in a news post last year, and to copy-paste the wording devoid of that context and without any news sounds off

```suggestion
22 of last year's songs will also later be released as part of an album. To get a feeling of what this tournament has to offer sound-wise, here's a sample:
```

<div align="center" class="osu-md__paragraph">
    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://www.youtube.com/embed/uA8mW0SXhxg" frameborder="0" allowfullscreen></iframe>
</div>

> Cannot get enough of Resurrection cup? Make sure to check out the [maps and songs in question](/wiki/osu!_originals#resurrection-cup-2024).

Reviewer: mm sounds more natural

```suggestion
Can't get enough of Resurrection cup? Make sure to check out the [maps and songs in question](/wiki/osu!_originals#resurrection-cup-2024).
```

## What's new?

While Resurrection Cup mostly remains the same as what you are used to, we are changing things up a tiny bit. This year, instead of 4v4, Resurrection Cup will feature a 3v3 format with teams of up to 5 players! Furthermore, all players in a team are required to be from the same geographic region. More on that on our [wiki page](/wiki/Tournaments/RESC/2025)!

> As to be expected from us, we have a lot of suprises up our sleeves this time around as well. With a legion of new mappers and artists joining our ranks, we are sure to bring you countless memorable original songs and many, many custom beatmaps designed and fine-tuned by experienced mappers, mappoolers and playtesters.

Reviewer 2: ```suggestion
As to be expected from us, we have a lot of surprises up our sleeves this time around as well. With a legion of new mappers and artists joining our ranks, we are sure to bring you countless memorable original songs and many, many custom beatmaps designed and fine-tuned by experienced mappers, mappoolers and playtesters.
```

You can head over to the [website](https://rescup.xyz/) to find out more details about the overall mappool structure as well as other relevant rules.

## Tournament schedule

| Event | Dates |
| --: | :-- |
| Registration phase | May 11 – June 7 (23:59 UTC) |
| Qualifier showcase | May 18 (17:00 UTC) |
| Qualifier stage | May 18 – June 7 |
| Screening phase | June 7 – June 14 |
| Round of 32 | June 20 – June 22 |
| Round of 16 | June 27 – June 29 |
| Quarterfinals | July 4 – July 6 |
| Semifinals | July 11 – July 13 |
| Finals | July 18 – July 20 |
| Grand Finals | July 25 – July 27 |

## How do I register?

To register, head over to the [website](https://rescup.xyz/) to connect your osu! and Discord account, as well as setting up your team! Before registering, please make sure that:

- You have not have violated the [osu! community rules](/wiki/Rules) within the past 12 months.
- You have joined the [ResCup25 official Discord server](https://discord.com/invite/UNzyfgGfeu).

> On top of that, you are required to sign up with a team of at least 3 members. If you don't have enough team members, the `#looking-for-team` channel in the [Discord server](https://discord.gg/UNzyfgGfeu) should be where you are heading first before signing up!

Reviewer: redirect can be removed to save time on click

```suggestion
On top of that, you are required to sign up with a team of at least 3 members. If you don't have enough team members, the `#looking-for-team` channel in the [Discord server](https://discord.com/invite/UNzyfgGfeu) should be where you are heading first before signing up!
```
